mill-build/src/millbuild/Deps.scala

```diff
@@ -307,7 +307,7 @@ object Deps {
     val uiTooling = mvn"androidx.compose.ui:ui:1.7.6"
     val screenshotValidationJunitEngine =
       mvn"com.android.tools.screenshot:screenshot-validation-junit-engine:0.0.1-alpha09"
-
+    val cmdlineToolsVersion = "19.0"
```

Comment:
This should be the version mill uses to do the initial setup but https://github.com/com-lihaoyi/mill/pull/5608/files#diff-02a26e3d4f6a4df457e3e14bc7da46644c3b6be0d7c95a8d04ff7b5eb53c4a17R298 is hardcoded